include/ck/tensor_operation/gpu/device/device_cgemm.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <iostream>
+#include <vector>
```

Comment:
You don't use those headers here. Just `<memory>` is enough.

Reply:
Fixed, device_base.hpp is enough. Done in Added in all new or modified hpp/cpp files in [ee06099](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/230/commits/ee060994463e472cf64010d87f5ba3fc3b0c624e)